package.yaml

```diff
@@ -33,8 +33,8 @@ dependencies:
   - optics-core
   - optics-th
   # PureSMT deps
-  - process
-  - typed-process
+  - inline-c
+  - z3
```

Comment:
This line should be removed as well if we truly want to remove the dependency on `haskell-z3`, otherwise `test/run-tests.sh` will repopulate `pirouette.cabal` with the dependency. But when I do so the build fails again with
```
Configuring test suite 'spec' for pirouette-2.0.0..
Preprocessing test suite 'spec' for pirouette-2.0.0..
Building test suite 'spec' for pirouette-2.0.0..
[ 1 of 13] Compiling Language.Pirouette.ExampleSpec ( tests/unit/Language/Pirouette/ExampleSpec.hs, /home/qaristote/code/pirouette/dist-newstyle/build/x86_64-linux/ghc-9.0.2/pirouette-2.0.0/t/spec/build/spec/spec-tmp/Language/Pirouette/ExampleSpec.o, /home/qaristote/code/pirouette/dist-newstyle/build/x86_64-linux/ghc-9.0.2/pirouette-2.0.0/t/spec/build/spec/spec-tmp/Language/Pirouette/ExampleSpec.dyn_o )
<command line>: /home/qaristote/code/pirouette/dist-newstyle/build/x86_64-linux/ghc-9.0.2/pirouette-2.0.0/build/libHSpirouette-2.0.0-inplace-ghc9.0.2.so: undefined symbol: Z3_mk_config
[FAILURE] 'cabal build' failed, will exit with 1
```